fastlane/Fastfile

```diff
@@ -11,69 +11,69 @@ configured = false
 is_build_pr = false
 
 # Executes before anything else use to setup the script
-before_all do |lane, options|
-  if lane.to_s == 'build_pr'
-    pr = options[:pr]
-    UI.success("Building #{pr}")
-    ENV['BRANCH_TO_BUILD'] = pr
-    is_build_pr = true
-  end
+before_all do
+  is_build_pr = ENV['BUILD_PR'] == 'true'
+  UI.success("Building for release #{ENV['BUILD_FOR_RELEASE']}")
 
-  # Raises an error is git is not clean
-  if ENV['COMMIT_CHANGES_TO_GIT'] == 'true'
-    ensure_git_status_clean
-  end
+  if ENV['CIRCLECI'] != 'true'
+    # Raises an error is git is not clean
+    if ENV['COMMIT_CHANGES_TO_GIT'] == 'true'
```

Comment:
why would circle have something to commit to git?

Reply:
npm install can sometimes change the packge-lock.json not with versions of the packages but all the `optional`, `developer` and those sort of flags thus marking the branch has dirty and this will then fail the whole build.